base/pkg/entry.jl

```diff
@@ -376,7 +376,11 @@ function update(branch::AbstractString)
                         LibGit2.fetch(repo)
                         LibGit2.merge!(repo, fastforward=true)
                     catch err
-                        show(err)
+                        if isa(err, LibGit2.GitError)
+                            print("Package $pkg cannot be updated. Error: $(err.msg) Class: $(err.class)")
+                        else
+                            show(err)
```

Comment:
As usual truth somewhere in between. I'm going to print exceptions which are clearly come from the package manager functionality and we aware of, such as `PkgError`, `GitError`. The rest will be rethrown.